datajoint/settings.py

```
@@ -248,6 +248,9 @@ def __setitem__(self, key, value):
                 self._conf[key] = value
             else:
                 raise DataJointError("Validator for {0:s} did not pass".format(key))
+            valid_logging_levels = ["DEBUG", "INFO", "WARNING", "ERROR", "CRITICAL"]
+            if "loglevel" in key and value in valid_logging_levels:
```

```suggestion
            if "loglevel" == key and value in valid_logging_levels:
```

---

should probably raise an error otherwise?